restructure.yml

```diff
@@ -79,6 +79,10 @@ worker:
   numThreads: 2
   # Maximum number of files to process in any given topic.
   maxFilesPerTopic: null
+  # Minimum time in seconds since a source file was last modified before it is
+  # considered for processing. This avoids synchronization issues when a file has just been
+  # written.
+  minimumFileAge: 60
```

Comment:
I thought we would have this setting for cleaner process. It makes sense to have it in the worker process as well. But would it also be helpful to have it on the cleaner process (not to check if the file is old enough or fully processed, until it has met certain age)?

Reply:
Ah aha, my aim was to avoid missing records in the first place. For the cleaner script, the following if a file is considered not fully extracted, its offsets will be removed from the accountant. It will not be checked again by the cleaner, until the restructure worker has re-extracted the file. By that time, the restructure worker should either successfully extract, or fail. In the former case, the file can be cleaned. In the latter, it will keep trying to unsuccessfully extract the file. There is no measure against that yet, at the moment.

Reply:
I see. I think I understand it better.